sections/exchange/hooks/useExchange.tsx

```diff
@@ -814,7 +814,8 @@ const useExchange = ({
 			}
 		}
 		return null;
-	}, [totalTradePrice, estimatedBaseTradePrice, txProvider]);
+		// eslint-disable-next-line
+	}, [estimatedBaseTradePrice, txProvider]);
```

Comment:
`totalTradePrice` in the dep array was causing bad slippage calcs while new `estimatedBaseTradePrice` data was loading